src/planscape/forsys/views.py

```diff
@@ -41,16 +45,6 @@ def convert_dictionary_of_lists_to_rdf(lists):
   rdf = rpy2.robjects.vectors.DataFrame(data)
   return rdf
 
-# Translates polygon.
-def translate_polygon(polygon, dx, dy):
-  coords = []
-  for ring in polygon.coords:
-    for point in ring:
-      coords.append((point[0] + dx, point[1] + dy))
-  translated = Polygon( tuple(coords) ) 
-  translated.srid = polygon.srid
-  return translated
-
 def get_boundary_debug_info(boundaries, project_area):
```

Comment:
Can you add type annotations to the parameters and for the output?

Also, this is a big function.  Can it be broken up at all?

Reply:
type annotations: done (though I want to go through them a second time to see whether I can make the annotations more strict).

big function: which function do you mean? I'm guessing you didn't intend to highlight "get_boundary_debug_info".